nle/nethack/nethack.py

```diff
@@ -77,10 +77,24 @@ def _set_env_vars(options, hackdir, wizkit=None):
         os.environ["WIZKIT"] = os.path.join(hackdir, wizkit)
 
 
+def tty_render(chars, colors):
```

Comment:
could you add a (optional) cursor tothis? I believe cursor can be rendered with an underline effect?

Reply:
or  reverse?

Reply:
Nice idea. Done.

Reply:
![image](https://user-images.githubusercontent.com/69371/126661893-3d949402-98ef-439b-8586-246bc7cc842d.png)

Reply:
It underlines the `f` a little bit because the cat and the hero are so close.